R/compat-dplyr.R

```diff
@@ -57,3 +57,12 @@ map_dfr <- function(.x, .f, ...) {
   res <- map(.x, .f, ...)
   vec_rbind(!!!res)
 }
+
+mutate <- function(.data, ...) {
```

Comment:
use base::transform() instead? use case here seems like a drop-in

Reply:
ok sure.